src/shogun2-core/src/main/java/de/terrestris/shogun2/service/GeoServerInterceptorService.java

```diff
@@ -271,14 +271,26 @@ private InterceptorRule getMostSpecificRule(String requestService,
 		Predicate<InterceptorRule> condition = new Predicate<InterceptorRule>() {
 			@Override
 			public boolean evaluate(InterceptorRule rule) {
-				// most specific: we have a rule with a matching endPoint
-				//                and operation
+				// most specific: we have a rule with a matching endPoint, service and operation
 				if (Objects.equals(rule.getEndPoint(), endPoint) &&
+						Objects.equals(rule.getOperation(), operation) &&
+						Objects.equals(rule.getService(), service)) {
+					LOG.trace("  * " + rule + " is endPoint, service and operation specific.");
+					return true;
+				// operation specific: if we have a rule with a matching endPoint and service,
+				// but not with a matching operation
```

Comment:
Above comment is misleading, or am I reading the code wrong?

Reply:
Alright, fixed in 89d6f6e21